filters/quantitative_ques/README.md

```diff
@@ -0,0 +1,17 @@
+## Quantitative question filter
+
+## What type of a filter is this?
+
+This filter is used to identify the quantitative questions
+
+## Why is measuring performance on this split important?
+This filter can be used to sepearte out quantiative questions, it helps to analyse models performance on questions which require numerical understanding.It is also usefull to study possible biases in question generation.
+
+python evaluate.py -f TextContainsKeywordsFilter -p=1  
+Here is the performance of the model on the filtered set
+The accuracy on this subset which has 219 examples= 95.89041095890411
```

Comment:
It would be instructive and useful to know and document the performance of the model on the entire [unfiltered] set as well, so that we know the effect and extent to which this filter is making things easier/harder.

Also, would be nice if you explicitly note here which dataset's test set you filtered to get the number above.

Reply:
Thanks for the review  @vgtomahawk 

I will update the readme with the scores and evaluation details

Reply:
Evaluation Results
default model and datasets were used : `mrm8488/bert-tiny-5-finetuned-squadv2`, validation[:20%] split of the `squad`

| Question type | Number of samples| EM|
|--------------------|------|--------|
|All| 2114| 60.31|
| Quant | 230| 70.86|
|Non Quant|1884| 59.02|

Results hint towards the fact that non quantitative questions are more challenging hence measuring performance along this split is important  <br>
I have made some code changes to make the filter compatible with the evaluation script will update all these in the next commit

Thanks